specification/postgresql/resource-manager/Microsoft.DBforPostgreSQL/preview/2022-03-08-preview/FlexibleServers.json

```diff
@@ -97,6 +97,9 @@
               "$ref": "#/definitions/Server"
             }
           },
+          "202": {
+            "description": "Accepted"
+          },
```

Comment:
Create with PUT shouldn't return 202 Accepted. You're recommended to use provisioningState or async operation header instead in the RPC.

Reply:
[https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/async-api-reference.md#creating-or-updating-resources-asynchronously](https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/async-api-reference.md#creating-or-updating-resources-asynchronously)